content/blog/2023/09/15/2023-09-15-jenkins-august-newsletter.adoc

```diff
@@ -0,0 +1,88 @@
+
+Contributed by: author:wadeck[Wadeck Follonier]
+
+* Andrea Chiera completed his 3 months internship within the Security team, auditing 100 plugins and finding 20+ vulnerabilities
+** link:/blog/2023/08/23/summer-internship-in-jenkins-security/[Summer Internship in Jenkins security ]
+** Thank you very much for your involvement and also to the team for mentoring him
+* Plugin security advisory published on July 12
+** link:/security/advisory/2023-08-16/[Jenkins Security Advisory 2023-08-16]
+** Four high score vulnerabilities, several medium and lower.
+** A total of 13 plugins were affected
+* The Security team is progressing on the CSP compliance project for Jenkins Core, encountering interesting corner cases that have to be covered.
+** link:https://issues.jenkins.io/browse/JENKINS-71014[https://issues.jenkins.io/browse/JENKINS-71014]
+
+[[Governance]]
+image:/images/post-images/2023/01/12/jenkins-newsletter/governance.png[Governance Update]
+
+Contributed by: author:markewaite[Mark Waite]
+
+[[infrastructure]]
+image:/images/post-images/2023/01/12/jenkins-newsletter/infrastructure.png[Infrastructure Update]
+
+Contributed by: author:dduportal[Damien Duportal]
+
+* JDK21 (Early Available Temurin Edition) and Maven 3.9.4 generally available for developers on ci.jenkins.io
+* JDK17 is the default for all the Jenkins controllers and agents on the Jenkins public infrastructure
+* Migration of the 2 last VMs to a new IPv6-enabled network: removed worth 1k$ (monthly) of unused cloud resources
+
+[[modern-ui]]
+image:/images/post-images/2023/01/12/jenkins-newsletter/ui_ux.png[User Experience Update]
+
+Contributed by: author:markewaite[Mark Waite]
+
+[[platform]]
+image:/images/post-images/2023/01/12/jenkins-newsletter/platform-modernization.png[Platform Modernization Update]
+
+Contributed by: author:gounthar[Bruno Verachten]
+
+* There has been an issue with Docker Images republishing unexpectedly, causing old tags to be rebuilt. This is now solved.
+* JDK21:
+** Work is ongoing to keep the JDK21 updated to an early access version in Jenkins’ infrastructure.
+** Jenkins can run on JDK21 from version 2.419
+** link:https://docs.google.com/document/d/1y3RVlniNmz-5Nd3LI-w58LDf760Ai7FqssP4zHuTv8U/edit?usp=sharing[+++A proposal from Mark Waite+++] regarding Java 11, Java 17, and Java 21 adoption roadmap in Jenkins is being considered.
+* Docker agents and controller:
+** Ssh-agent: breaking change in 5.10.0 with the replacement of bullseye in favor of link:https://github.com/jenkinsci/docker-ssh-agent/pull/299[bookworm].
+** Controller:
+*** 2.414.1 brought a breaking change: link:https://github.com/jenkinsci/docker/pull/1687[update debian images to bookworm]
+** Windows:
+*** Windows agent images are now using a Windows serve image, which is a breaking change.
+*** Windows 2022 images are being considered due to Windows 2019 reaching end-of-life.
+*** Instructions for Java 17 are still to be updated in the Windows docs.
```

Comment:
```suggestion
* There has been an issue with Docker Images republishing unexpectedly, causing old tags to be rebuilt.
This is now solved.
* JDK21:
** Work is ongoing to keep JDK21 updated to an early access version in Jenkins’ infrastructure.
** Jenkins can run on JDK21 since version 2.419.
** link:https://docs.google.com/document/d/1y3RVlniNmz-5Nd3LI-w58LDf760Ai7FqssP4zHuTv8U/edit?usp=sharing[+++A proposal from Mark Waite+++] regarding Java 11, Java 17, and Java 21 adoption roadmap in Jenkins is being considered.
* Docker agents and controller:
** Ssh-agent: breaking change in 5.10.0 with the replacement of bullseye in favor of link:https://github.com/jenkinsci/docker-ssh-agent/pull/299[bookworm].
** Controller:
*** 2.414.1 brought a breaking change: link:https://github.com/jenkinsci/docker/pull/1687[update debian images to bookworm]
** Windows:
*** Windows agent images are now using a Windows server image, which is a breaking change.
*** Windows 2022 images are being considered due to Windows 2019 reaching end-of-life.
*** Instructions for Java 17 are still to be updated in the Windows docs.
```
Some syntax adjustments, and I'm not sure if it is supposed to be "Windows serve image" or "Windows server image". I suggested it to raise attention, but if it is intended to be `serve` then disregard!